### PR TITLE
WEBEU2 - Preprocessing II - Rui Hildt 🇧🇪

### PR DESCRIPTION
**Pull request by ruihildt**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,29 +4,29 @@ Fun Bus is a travel agency looking for some help on their website.  They want a
 
 ## Task 1: Set Up The Project With Git
 
-- [ ] Create a forked copy of this project.
-- [ ] Add your project manager as collaborator on Github.
-- [ ] Clone your OWN version of the repository (Not Lambda's by mistake!).
-- [ ] Create a new branch: git checkout -b `<firstName-lastName>`.
-- [ ] Implement the project on your newly created `<firstName-lastName>` branch, committing changes regularly.
-- [ ] Push commits: git push origin `<firstName-lastName>`.
+- [x] Create a forked copy of this project.
```
Comment (maximesalomon) [inline README.md]: Great job using the README

File: index.html [hunk 2]
```diff
@@ -3,9 +3,8 @@
 <html lang="en">
 <head>
   <meta charset="utf-8">
-
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
```
Comment (maximesalomon) [inline index.html]: 👍

File: index.html [hunk 3]
```diff
@@ -16,10 +15,27 @@
...
+        <a href="">About Us</a>
+        <a href="">Blog</a>
+        <a href="">Contact</a>
+      </nav>
+    </div>
+  </header>
+
   <div class="container home">
-   
+    <section class="main-header" >
```
Comment (maximesalomon) [inline index.html]: no `<main>`?

File: less/variables.less [hunk 4]
```diff
@@ -1,7 +1,7 @@
 
 // Color assignments 
 @font-color: @shark;
-@button-bg: @eastern-blue;
+@button-bg: @coolblue;
```
Comment (maximesalomon) [inline less/variables.less]: :)

File: pr.md [hunk 5]
```diff
@@ -0,0 +1 @@
+WEBEU2 - Preprocessing II - Rui Hildt 🇧🇪
```
Comment (maximesalomon) [inline pr.md]: How do you use this?

File: index.html (hunk 3 above)
Comment (ruihildt) [reply index.html]: That was the code provided, so I figured it was the way to go.

File: pr.md (hunk 5 above)
Comment (ruihildt) [reply pr.md]: When I create the PR with github CLI (HUB), I have to provide an md file with the title as content, I should probably remove it after submitting the PR.

Here's my command `hub pull-request -c -F pr.md`.

so I will try `hub pull-request -c -F pr.md && rm pr.md`

**Comment by ruihildt** [comment]
@maximesalomon 

> A small thing you could improve is add an Uppercase to your commit messages haha
> 
> add --> Add :)

I remembered the opposite from reading the commit formatting article, I'll make sure to do it from next week. (it's not in the Sprint Challenge, as I read your comments after)

**Pull request merged**